main.py

```diff
@@ -54,6 +55,9 @@ def get_args():
 
     specParser = SpecParser(**vars(args))
     spec = specParser.parse(args.spec_dir)
+    if isError():
+        logging.warning(f"Spec not parsed successfully.")
```

Comment:
NIT: I think, this could be an error log

Reply:
changed and force-pushed